projects/1.20/assets/powah-rearchitected/powah/lang/zh_cn.json

```
@@ -0,0 +1,265 @@
+    "block.powah.magmator_blazing": "岩浆发电机（烈焰）",
+    "block.powah.magmator_hardened": "岩浆发电机（硬化）",
+    "block.powah.magmator_niotic": "岩浆发电机（钻石）",
+    "block.powah.magmator_nitro": "岩浆发电机（下界）",
+    "block.powah.magmator_spirited": "岩浆发电机（富生）",
+    "block.powah.magmator_starter": "岩浆发电机（初级）",
+    "block.powah.niotic_crystal_block": "钻石晶体块",
+    "block.powah.nitro_crystal_block": "下界晶体块",
+    "block.powah.player_transmitter_basic": "玩家供电仪（基础）",
+    "block.powah.player_transmitter_blazing": "玩家供电仪（烈焰）",
+    "block.powah.player_transmitter_hardened": "玩家供电仪（硬化）",
+    "block.powah.player_transmitter_niotic": "玩家供电仪（钻石）",
+    "block.powah.player_transmitter_nitro": "玩家供电仪（下界）",
+    "block.powah.player_transmitter_spirited": "玩家供电仪（富生）",
+    "block.powah.player_transmitter_starter": "玩家供电仪（初级）",
+    "block.powah.reactor_basic": "反应堆（基础）",
+    "block.powah.reactor_blazing": "反应堆（烈焰）",
+    "block.powah.reactor_hardened": "反应堆（硬化）",
+    "block.powah.reactor_niotic": "反应堆（钻石）",
+    "block.powah.reactor_nitro": "反应堆（下界）",
+    "block.powah.reactor_spirited": "反应堆（富生）",
+    "block.powah.reactor_starter": "反应堆（初级）",
+    "block.powah.solar_panel_basic": "太阳能板（基础）",
+    "block.powah.solar_panel_blazing": "太阳能板（烈焰）",
+    "block.powah.solar_panel_hardened": "太阳能板（硬化）",
+    "block.powah.solar_panel_niotic": "太阳能板（钻石）",
+    "block.powah.solar_panel_nitro": "太阳能板（下界）",
+    "block.powah.solar_panel_spirited": "太阳能板（富生）",
+    "block.powah.solar_panel_starter": "太阳能板（初级）",
+    "block.powah.spirited_crystal_block": "富生晶体块",
+    "block.powah.thermo_generator_basic": "热力发电机（基础）",
+    "block.powah.thermo_generator_blazing": "热力发电机（烈焰）",
+    "block.powah.thermo_generator_hardened": "热力发电机（硬化）",
+    "block.powah.thermo_generator_niotic": "热力发电机（钻石）",
+    "block.powah.thermo_generator_nitro": "热力发电机（下界）",
+    "block.powah.thermo_generator_spirited": "热力发电机（富生）",
+    "block.powah.thermo_generator_starter": "热力发电机（初级）",
+    "block.powah.uraninite_block": "晶质铀块",
+    "block.powah.uraninite_ore": "晶质铀矿石",
+    "block.powah.uraninite_ore_dense": "晶质铀矿石（致密）",
+    "block.powah.uraninite_ore_poor": "晶质铀矿石（贫瘠）",
+    "chat.powah.no.binding": "这张卡已被绑定至%s！",
+    "chat.powah.not.enough.blocks": "方块不足，你还需要%s个！",
+    "chat.powah.wrench.link.done": "链接完成！",
+    "chat.powah.wrench.link.fail": "超出范围！",
+    "chat.powah.wrench.link.start": "开始链接……",
+    "gui.powah.jei.category.coolant": "冷却剂",
+    "gui.powah.jei.category.energizing": "充能",
+    "gui.powah.jei.category.heat.sources": "热源",
+    "gui.powah.jei.category.magmatic": "岩浆流体",
+    "gui.powah.jei.category.reactor.fuels": "反应堆燃料",
+    "gui.powah.jei.category.solid.coolant": "固体冷却剂",
+    "info.powah.carbon": "碳",
+    "info.powah.channel": "末影频道：%s",
+    "info.powah.charging.speed": "充能速度：%s FE/t",
+    "info.powah.click.to.bind": "右击以绑定。",
+    "info.powah.fuel.consumption": "燃料消耗",
+    "info.powah.gen.mode": "自动模式",
+    "info.powah.gen.mode.desc": "反应堆会在能量缓存满的时候停止\n并在能量少于70%时启动",
```

| **审阅级别** |
|:--------------:|
|⚪ **轻微** / 不阻止合并但需三思|

原文：
**The reactor will stop when is full and start 
when has less than 70% of energy.**
```suggestion
    "info.powah.gen.mode.desc": "反应堆会在能量缓存满的时候停止\n并在能量少于70%时启动。",
```